ansible/roles/nodepool/files/elements/jenkins-slave/package-installs.yaml

```diff
@@ -10,3 +10,4 @@ redhat-lsb-core:
 redhat-rpm-config:
 vim:
 wget:
+docker:
```

Comment:
Can you please sort the packages by name?

Reply:
sure